src/modules/shared/components/ListNavigation/ListNavigation.tsx

```diff
@@ -28,7 +30,11 @@ const ListNavigation: FC<ListNavigationProps> = ({
 									item.active && styles['c-list-navigation__item--active']
 								)}
 							>
-								<div style={{ paddingLeft: `${layer * 3.2}rem` }}>{item.node}</div>
+								<div style={{ paddingLeft: `${layer * 3.2}rem` }}>
+									{typeof item.node === 'function'
+										? item.node(nodeProps)
+										: item.node}
```

Comment:
Waarom is dit nodig? `styles` is toch ook importeerbaar in andere files?

Reply:
Ik vond het zelf raar om vanuit een page of andere component de styles te moeten binnentrekken van de ListNavigation om het werkende te krijgen, dit zou gwn vanuit de component moeten gebeuren.
Zonder al te grote aanpassingen te moeten doen was dit het eenvoudigste om te implementeren.